### PR TITLE
v4.9.2

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Ref #30

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
-{% set version = "4.9.3" %}
-{% set build_number = 1 %}
+{% set version = "4.9.2" %}
+{% set build_number = 2 %}
```
Comment (BastianZim) [inline recipe/meta.yaml:2]: Highest build number of both outputs is currently 1.

**Comment by BastianZim** [comment]
@bollwyvl Would you mind doing your magic with the patch again?

**Comment by bollwyvl** [comment]
Maybe it will work :shrug: ?

**Comment by BastianZim** [comment]
Works, thanks!

**Comment by BastianZim** [comment]
Do you want to merge this into a different branch than `master`?

**Comment by bollwyvl** [comment]
> different branch

Sure, we can make a quick temporary branch...

**Comment by bollwyvl** [comment]
>  temporary branch

I guess we can keep it around in case we need to do more archaeology...

**Pull request merged**